pkg/pipeline/ingest/ingest_kafka_test.go

```diff
@@ -177,9 +173,4 @@ func Test_KafkaListener(t *testing.T) {
 
 	require.Equal(t, 1, len(receivedEntries))
 	require.Equal(t, string(fakeRecord), receivedEntries[0])
-
-	// make the ingest thread exit
-	close(ingestKafka.exitChan)
```

Comment:
@mariomac This might cause failure in e2e test

Reply:
@mariomac I reran the e2e and I think they failed due to glitch in github networking (access and not due to this) but we need to keep eyes open.

Reply:
Yeah it looked like an issue with networking but let's keep an eye.

In theory, when the test is finished the channels should be closed automatically by the runtime. We can do this operation explicit but it would make the test setup a bit less clear.